packages/pilot/src/pages/Account/index.js

```diff
@@ -39,6 +40,10 @@ const AccountArea = ({ t }) => (
             component={Login}
             path="/account/login"
           />
+          <Route
```

Comment:
```suggestion
{impersonate && (
  <Route
    component={LoginImpersonate}
    path="/account/impersonate"
  />
)}
          <Route
```

Assim evita até de subir a rota em prod e sandbox.

Reply:
Boa! Acho super válido. Fazendo isso agora também